MoEngagePluginBase/MoEngageCardsPluginManager.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MoEngageCardsPluginHandler.swift
+//  MoEngagePlugin
+//
+//  Created by Rakshitha on 16/08/23.
+//  Copyright © 2022 MoEngage. All rights reserved.
+//
+
+import Foundation
+
+class MoEngageCardsPluginManager: NSObject {
```

Comment:
Can we generalize this for any plugin? Ideally Core shouldn't be aware of dependant modules and we shouldn't have to release core when onboarding new modules.

Reply:
@msoumya-engg-sdk done